apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -310,9 +316,10 @@ export class SendMessageEmail extends SendMessageType {
         })
       );
 
-      if (!result.id) {
+      if (!result?.id) {
```

Comment:
This broke a test. But that also means that sending the mail doesn't return anything anymore. Worth to double check.